connectors/src/lib/data_sources.ts

```diff
@@ -27,8 +27,9 @@ import { statsDClient } from "@connectors/logger/withlogging";
 import type { DataSourceConfig } from "@connectors/types/data_source_config";
 
 const axiosWithTimeout = axios.create({
-  timeout: 60000,
+  timeout: 120000,
```

Comment:
This seems like a lot, are we sure about this? It seems weird that this page passes the check on the content length but still fail to be updated within 60 seconds.

Reply:
I checked locally and it passes the content length, it's 710863 while limit is set to 750000.
I imagined I could either increase the timeout or reduce the content length and I opted for the first. 
Do you think we should not touch the timeout and opt for the other option?

Reply:
Having queries that are 2 minutes long somewhat contradicts our current infrastructure. The longer the query, the higher the likelihood it will be terminated during deployment. We do have a brief grace period during deployment, but the pod will certainly be terminated around 90 seconds after the signal is sent. Reducing the content size does not seem like a bad idea.

Reply:
@flvndvd I added a commit, WDYT?